### ISSUE TITLE
Search bar on top of components list.

### ISSUE THREAD

**Issue by VasudevHarshal-123** [ampproject/amp.dev#3685]
### Problem
Amp documentation has a long list of components and they are arranged in alphabetical order with alphabetical headings which is really helpful, but, If someone wants to see for a component or check if what they are looking for exists or not, they need to scroll down to the bottom in the worst-case scenario.
The list of components will definitely increase with time. So, having a search bar will help.   

### Solution
A search bar on top of the components list to help the user search through amp components.

### Alternatives
Ctrl+F is an alternative but a search bar would be really great.

**Comment by sebastianbenz** [comment]
Yeah, makes sense. You can use the site search at the top, which offers direct auto completion for components, but a combo-box like search for components could be helpful.
